### DISCUSSION TITLE
WIKI FORMAT - Brewing, Craftings and Enchanting Missions.

### DISCUSSION THREAD

**Discussion by JLucassz** [BG-Software-LLC/SuperiorSkyblock2#759] [Pending Feature Requests] [answered]
Hi, i need help, how should i format BrewingMissions, CraftingsMissions and EnchantingMissions ? It would be helpful, because i can't find it anywhere.

**Comment by Lemvs** [comment, marked as answer]
Wiki contains that informations for a long time: https://wiki.bg-software.com/superiorskyblock/overview/missions

@OmerBenGera you can mark this answered and close it.
